### PR TITLE
Feat: Add descriptions utils for evm scripts and transactions

### PR DESCRIPTION
**Pull request by 0xGabi**
This PR includes a utility to describe EVM scripts and transactions. 

It exposes a few of the utils we use on `connect-core` to be accessible from `connect`. 

It includes an example of how to use `decodeScript` to decode a script from a vote.

### CODE REVIEW

File: examples/nodejs/src/describe-script.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { connect, describeScript, App } from '@aragon/connect'
```
Comment (0xGabi) [inline examples/nodejs/src/describe-script.ts:1]: Example of how to use `describeScript` to decode script from votes.

File: packages/connect-core/src/utils/index.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export { findAppMethodFromIntent } from './app'
```
Comment (0xGabi) [inline packages/connect-core/src/utils/index.ts:1]: I decided to expose this utilities, what you think?

File: packages/connect-core/src/utils/descriptions.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+ */
+export async function describeTransactionPath(
+  path: TransactionWithChildren[],
+  apps: App[],
+  provider?: ethers.providers.Provider
+): Promise<TransactionRequest[]> {
+  return Promise.all(
+    path.map(async step => {
+      return describeTransaction(step, apps, provider)
+    })
```
Comment (bpierre) [inline packages/connect-core/src/utils/descriptions.ts:66]:
```suggestion
    path.map(step => describeTransaction(step, apps, provider))
```

File: packages/connect-core/src/utils/index.ts (hunk 2 above)
Comment (bpierre) [reply packages/connect-core/src/utils/index.ts:1]: Sounds good yes!

**Pull request merged**